compiler/src/test/java/com/squareup/anvil/compiler/dagger/BindsMethodValidatorTest.kt

```diff
@@ -54,8 +56,11 @@ class BindsMethodValidatorTest(
       )
       if (!useDagger) {
         assertThat(messages).contains(
-          "Expected binding of type Bar but impl parameter of type Foo only has the following " +
-            "supertypes: [Ipsum, Lorem]",
+          Errors.bindsParameterMustBeAssignable(
```

Comment:
I'd prefer we keep the hardcoded expected value in this case, since it's possible a bug could be introduced in `bindsParameterMustBeAssignable()` later on which automatically changes the test to expect an incorrect value.

Reply:
Fixed